assets/tests/TaskSchemaValidationServiceSpec.js

```diff
@@ -0,0 +1,301 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the QuestionSchemaValidatorService.
+ */
+
+describe('TaskSchemaValidationService', function() {
+  var QuestionDataService;
+  // TaskSchemaValidationService needs a shorter name.
```

Comment:
This raises more questions than it answers (my first question on reading it was "why?"). Suggest just writing

`// Alias for TaskSchemaValidationService.`